sos/policies/redhat.py

```diff
@@ -412,13 +421,15 @@ class CentOsAtomicPolicy(RedHatAtomicPolicy):
     distro = "CentOS Atomic Host"
     vendor = "CentOS"
     vendor_url = "https://www.centos.org/"
+    id = "centos"
```

Comment:
This would never be evaluated as this patch intends. `CentOsAtomicPolicy` subclasses `RedHatAtomicPolicy` which overrides the `check()` method of `RHELPolicy` where the changes for checking for the `id` field is being made.

I don't think we need to add an `id` check here anyways, for two reasons. 1) The current `check()` is looking for the string `Atomic` in the release file which should be sufficient. 2) Atomic Host is deprecated in favor of RHCOS, so I don't think we'll see too many rebuilds, much less those that change the release file.

Reply:
I agree, my intend was that it would still work if for some reason CentOS Atomic would again rely on content of /etc/os-release (the generic code)